pkgs/applications/science/electronics/verilog/default.nix

```diff
@@ -13,13 +23,6 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-sT9j/0Q2FD5MOGpH/quMGvAuM7t7QavRHKD9lX7Elfs=";
   };
 
-  enableParallelBuilding = true;
```

Comment:
Why is this redundant?

Reply:
A quick search led me to #32271 which sets ``enableParallelBuilding = true`` if using CMake, though TBH I didn't check if that line is still in nixpkgs.

Reply:
This is not a cmake build, so parallel building is not enabled by default.

Reply:
Ah, thanks. I'm mixing up projects in my head now. Was just working on a different one with cmake.